### PR TITLE
[admin] [ToC] Adding full application list to TOC

### PR DESCRIPTION
**Pull request by AlexJerabek**
Review site: https://review.docs.microsoft.com/en-us/javascript/api/overview?view=excel-js-preview&branch=AlexJ-FullToc

### CODE REVIEW

File: docs/docs-ref-autogen/visio/toc.yml [hunk 1]
```diff
@@ -73,3 +81,229 @@ items:
             uid: 'visio!Visio.ShapeMouseLeaveEventArgs:interface'
           - name: ShapeView
             uid: 'visio!Visio.ShapeView:class'
+      - name: Word
+        href: /javascript/api/word
+      - name: Common APIs
```
Comment (ElizabethSamuel-MSFT) [inline docs/docs-ref-autogen/visio/toc.yml:86]: I didn't think that Common APIs apply for Visio...?

Comment (AlexJerabek) [reply docs/docs-ref-autogen/visio/toc.yml:86]: They don't. I must have circumvented that check in this version.

**Comment by AlexJerabek** [comment]
@ElizabethSamuel-MSFT, Actually, hold up. The Visio scenario still isn't working as intended.

**Review by ElizabethSamuel-MSFT — APPROVED** [review]
Looks good to me

**Pull request merged**